docs/channels/category-heading.md

```diff
@@ -30,6 +30,20 @@ If no Categories are being shown, the tag will not show anything contained withi
 
 [TOC=3]
 
+### `category_id=`
+
+    category_id="5"
+
+The category_id= parameter allows you to override the detection of current category on URL.
+
+### `category_url_title=`
+
+    category_url_title="press-releases"
+
+The category_id= parameter allows you to override the detection of current category on URL without the use of the [Category URL Titles In Links](control-panel/settings/content-design.md) feature.
```

Comment:
"The category_id= parameter" should probably say `category_url_title=`? 
It might be good to also add backticks to have it better formatted